### DISCUSSION TITLE
Outdated and incorrect information about 2FA-reset on docs.github.com/

### DISCUSSION THREAD

**Discussion by safetytracker** [community/community#86621] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello.

I have a problem with login to second, personal github-profile, because I have lost my 2FA-app (device) and din't save "recovery-codes" file.

But on the page: https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#authenticating-with-a-verified-device-ssh-token-or-personal-access-token 

we can see words: "Use a recovery code or request a reset."
![image](https://github.com/community/community/assets/61080193/336eb40a-9983-4e3d-acbb-9c35829f3c60)

Unfortunately, there are no links on the Github site that could be clicked to reset the two-factor authentication using my email.

Please tell me, is this help-documentation already outdated and incorrect, or is there a way to reset the two-factor authorization?

(similar problem: https://github.com/orgs/community/discussions/70046 )

**Comment by ghostinhershell** [comment]
Hi @safetytracker, thanks so much for reaching out in our community. The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them.
